data/lenet_memory_solver.prototxt

```diff
@@ -1,9 +1,9 @@
 # The train/test net protocol buffer definition
 net: "lenet_memory_train_test.prototxt"
 # test_iter specifies how many forward passes the test should carry out.
-test_iter: 1
+test_iter: 2
```

Comment:
We have to have a separate copy of prototxt if it needs to be used from resources......the path in solver for train_test.prototxt is relative which is required my InterleaveTest.scala whereas in python we do a spark-submit and ship both the prototxt files .....which gets copied in the current dir of executor context. If solver points to train_test.prototxt relative path....it is a problem. So the path of train_test.prototxt should be current dir. 
This is the reason we do most of things in data. Anyway I can create a separate copy of solver prototxt for python